tools/cp-caps/README.md

```diff
@@ -66,6 +66,7 @@ options:
   -f, --frame-format   test whether the RCP supports 802.15.4 frames of all formats
   -p, --data-poll      test whether the RCP supports data poll
   -t, --throughput     test Thread network 1-hop throughput
+  -T, --tx-info        test mTxInfo field of the radio frame
```

Comment:
How about we structure test cases with [unittest](https://docs.python.org/3/library/unittest.html)?

Reply:
Will do it in another commit later.